client/components/catalog/TagFilter.vue

```diff
@@ -50,17 +50,18 @@ import find from 'lodash/find';
 import get from 'lodash/get';
 import map from 'lodash/map';
 import { mapState } from 'vuex';
+import orderBy from 'lodash/orderBy';
 
 export default {
   data: () => ({ search: '' }),
   computed: {
     ...mapState('repositories', ['tags', 'tagFilter']),
     isVisible: vm => get(vm.$refs.filter, 'isActive', false),
     options() {
-      return map(this.tags, it => {
+      return orderBy(map(this.tags, it => {
```

Comment:
Looks a bit too complex to read at the moment try to break it down and name orderByCond or similar ;) thx